tests/integration/api/v1/test_competition_endpoints.py

```
@@ -0,0 +1,573 @@
+        assert response.status_code == 200
+        assert response.json() == []
+
+    @pytest.mark.asyncio
+    async def test_list_competitions_with_data(self, client: AsyncClient):
+        """Listar competiciones retorna las creadas."""
+        user = await create_authenticated_user(
+            client, "lister2@test.com", "Pass123!", "List", "Two"
+        )
+
+        # Crear 2 competiciones
+        comp1 = await create_competition(client, user["token"])
```

Ruff (0.14.5)

Local variable `comp1` is assigned to but never used

Remove assignment to unused variable `comp1`

(F841)